package.json

```diff
@@ -7,7 +7,8 @@
   "license": "Apache-2.0",
   "scripts": {
     "clean": "rm -rf node_modules/ && npm cache clean --force",
-    "update-lock": "npm run clean && npm i --no-progress",
+    "update-dependencies": "npm run clean && npm i --no-progress",
+    "check-dependencies": "npm ls --depth=0 || npm run update-dependencies",
```

Comment:
```suggestion
    "check-dependencies": "npm ls --loglevel silent > /dev/null || npm run update-dependencies",
```

Reply:
Thing is, I'd like to see the dependencies that were having troubles (in case it fails) in the logs, since we might not be able to reproduce it locally. So better to have it with depth 0.

Reply:
But depth=0 is only checking for our dependencies not the peers ones ...

Reply:
Don't worry, the `--depth` will just modify the visual output, not the analysis itself ;)